docker-compose.yml

```diff
@@ -20,6 +20,34 @@ services:
       - /app/node_modules
     ports:
       - "8001:80"
+  
+  question-service-server:
+    build:
+      context: ./backend/question-service/server/app/
+      dockerfile: Dockerfile.dev
+    volumes:
+      - ./backend/question-service/server/app:/app:delegated
+    environment:
+      - MONGO_HOST=question-service-db
+      - MONGO_PORT=27017
+    depends_on:
+      question-service-db:
+        condition: service_healthy
+    ports:
+      - "8003:80"
+  
+  question-service-db:
+    image: mongo:latest
+    volumes:
+      - question-service-db-data:/data/db
+    ports:
+      - "27017:27017"
```

Comment:
i think this is fine for now but we need to eventually find a way to deconflict with other databases/ find another arrangement